src/components/Calender.vue

```diff
@@ -97,44 +106,47 @@
 		transition: background-color 1.6s ease-in;
 	}
 
-	a.date.padding
-	{
+	a.date.padding {
 		opacity: 0;
 		pointer-events: none;
 	}
 
 	a.date:hover,
-	a.date:focus
-	{
+	a.date:focus {
 		background-color: var(--red);
 		transition: background-color 0.3s ease;
 	}
-	
-	h3
-	{
+
+	h3 {
 		font-family: var(--numbers-font);
 		font-size: 2.5rem;
 		font-weight: normal;
 
-		padding: 0.22em 0.11em 0;
 		margin: 0;
 		margin-bottom: auto;
-		border-radius: 100%;
+		text-align: left;
```

Comment:
This leaves the numbers misplaced after the title disappears:
<img width="925" alt="Screen shot of the calendar after titles are removed" src="https://user-images.githubusercontent.com/19389660/197292320-49e740d0-529f-4583-8b0b-f374b9947e4a.png">